### DISCUSSION TITLE
Meaning of `total_market_value` field in clubs asset

### DISCUSSION THREAD

**Discussion by CarlosTjoeNij** [dcaribou/transfermarkt-datasets#164] [Q&A] [answered]
I was wondering from what year the 'total_market_value' from clubs.csv was extracted? Or is this always the most recent amount.
If it is possible I would like to get the total market value of each club for each year. Feel free to email me as well: carlostjoenij@gmail.com

Thanks in advance

**Comment by dcaribou** [comment, marked as answer]
Hi @CarlosTjoeNij,

I'm sorry for the late response.

The `total_market_value` value maps to that in [a club main page](https://www.transfermarkt.co.uk/fc-bayern-munchen/startseite/verein/27) in transfermarkt, which I believe corresponds to the current market value.

<img width="1050" alt="image" src="https://github.com/dcaribou/transfermarkt-datasets/assets/16071835/98128fac-be38-4403-a6a8-6656a0e0538d">


I don't recall transfermarkt having a record of historical values for a club market value that we can scrape from. If there's we can consider adding a new crawler in https://github.com/dcaribou/transfermarkt-scraper for it.

Another option could be to use the existing `player_valuations` asset (which does contain market value history), but you'd need figuring out historical membership of a player belonging to a club...

In summary, it does not sound easy 😄
